CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
++ Ensure both delays are written for the two SAR dates in the GUNW
```

Comment:
Correct this:
- Are we writing SAR delays int he GUNW? We are writing the interferometric hydrostatic and wet delays. 
- Also it more like at asset statement that captures failing XXX

Reply:
We are writing one netcdf file for each SAR date in the GUNW for subsequent processing. I''ll update to reflect that it's just an assert statement

Reply:
You mean you are having two intermediate netcdf files, these are then used to compute interferometric, and only interferometric is added to gunw?

Reply:
Yup